### ISSUE TITLE
doesn't work with python 3.9 on macbook pro m1 with conda install.

### ISSUE THREAD

**Issue by bayeslearner** [vatlab/sos#1505]
The bash kernel won't run correctly:
<img width="636" alt="image" src="https://user-images.githubusercontent.com/68746303/200408498-98a99ab6-5fdc-4dc4-9b15-19fad3356719.png">

```
/opt/homebrew/Caskroom/miniforge/base/envs/sos-env/lib/python3.9/site-packages/sos_notebook/kernel.py:1216: RuntimeWarning: coroutine 'ZMQSocketChannel.msg_ready' was never awaited
  while self.KC.shell_channel.msg_ready():
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
/opt/homebrew/Caskroom/miniforge/base/envs/sos-env/lib/python3.9/site-packages/sos_notebook/kernel.py:1217: RuntimeWarning: coroutine 'ZMQSocketChannel.get_msg' was never awaited
  self.KC.shell_channel.get_msg()
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
```

**Comment by BoPeng** [comment]
This is caused by incompatible versions of supporting libraries such as `pykernel`. Could you try to run `pip install XXX -U` for all jupyter related packages?

**Comment by bayeslearner** [comment]
i was using conda, not directly through pip.  Mixing it could break other packages.  Did you try?

**Comment by BoPeng** [comment]
What is the version of sos-notebook in conda? Did you try `conda updte -c conda-forge sos-notebook`?

**Comment by BoPeng** [comment]
I have just released sos-notebook 0.2.40 and new versions of all language modules. Please let me know if the problem persists after upgrading to these versions.
